libraries/kotlin.test/js/src/main/kotlin/kotlin/test/FrameworkAdapter.kt

```diff
@@ -11,13 +11,13 @@ package kotlin.test
  * The tests structure is defined using internal functions suite and test, which delegate to corresponding functions of a [FrameworkAdapter].
  * Sample test layout:
  *
- * suite('a suite', false, function() {
- *   suite('a subsuite', false, function() {
- *     test('a test', false, function() {...});
- *     test('an ignored/pending test', true, function() {...});
- *   });
- *   suite('an ignored/pending test', true, function() {...});
- * });
+ *     suite('a suite', false, function() {
```

Comment:
Perhaps it would be more reliable to use triple backtick surrounded block.

Reply:
@ilya-g Done.  It's now fenced in <code>\`\`\`js &hellip; \`\`\`</code> to make the language explicit.